infra/gcp/clusters/.gitignore

```diff
@@ -1,4 +1,16 @@
 .terraform/
-*.tfstate*
+.terraform.lock.hcl
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
```

Comment:
This matches both, unless there's a need to allow `*.tfstatefoo`?
```suggestion
*.tfstate*
```

Reply:
Fixed